code/modules/cargo/bounties/item.dm

```diff
@@ -1,14 +1,28 @@
 /datum/bounty/item
+	///How many items have to be shipped to complete the bounty
 	var/required_count = 1
+	///How many items have been shipped for the bounty so far
 	var/shipped_count = 0
-	var/list/wanted_types  // Types accepted for the bounty.
-	var/include_subtypes = TRUE     // Set to FALSE to make the datum apply only to a strict type.
-	var/list/exclude_types // Types excluded.
+	///Types accepted by the bounty (including all subtypes, unless include_subtypes is set to FALSE)
+	var/list/wanted_types
+	///Set to FALSE to make the bounty not accept subtypes of the wanted_types
+	var/include_subtypes = TRUE
+	///Types that should not be accepted by the bounty, also excluding all their subtypes
+	var/list/exclude_types
+	///Individual types that should be accepted even if their supertypes are excluded (yes, apparently this is necessary)
+	var/list/special_include_types
 
 /datum/bounty/item/New()
 	..()
-	wanted_types = typecacheof(wanted_types)
-	exclude_types = typecacheof(exclude_types)
+	wanted_types = typecacheof(wanted_types, only_root_path = !include_subtypes)
+	if (exclude_types)
+		exclude_types = typecacheof(exclude_types)
```

Comment:
Bounty datums are not singletons, are they?
```suggestion
		exclude_types = string_assoc_list(typecacheof(exclude_types))
```
If the list doesn't get modified this should provide a nice cache to diminish duplicates.